src/test/regress/bin/normalize.sed

```diff
@@ -40,6 +40,12 @@ s/ "citus_local_table_([0-9]+)_[0-9]+" / "citus_local_table_\1_xxxxxxx" /g
 # normalize relation oid suffix for the truncate triggers created by citus
 s/ truncate_trigger_[0-9]+ / truncate_trigger_xxxxxxx /g
 
+# (citus_table_triggers.sql)
+# postgres generates create trigger commands for triggers with:
+# "EXECUTE FUNCTION" in pg12
+# "EXECUTE PROCEDURE" in pg11
+s/EXECUTE PROCEDURE table_triggers_schema\.test_table_trigger_function/EXECUTE FUNCTION table_triggers_schema.test_table_trigger_function/g
```

Comment:
Isn't this part too specific `table_triggers_schema\.test_table_trigger_function`? what if the function name or schema name is different?

Reply:
actually that is right, I wanted to add a rule that's specific to these 3 test cases (https://github.com/citusdata/citus/pull/3995#discussion_r456287143)

another option can be `FOR EACH STATEMENT EXECUTE PROCEDURE` -> `FOR EACH STATEMENT EXECUTE FUNCTION`, what do you think ?

Reply:
or `EXECUTE PROCEDURE table_triggers_schema\.` -> `EXECUTE FUNCTION table_triggers_schema\.`, which would ensure that this rule will only apply to that file

Reply:
I think

`FOR EACH STATEMENT EXECUTE PROCEDURE -> FOR EACH STATEMENT EXECUTE FUNCTION`

seems good. That way we will have this normalization even if the schema/function_name changes. And this is only the case for trigger I guess so it doesn't cause any false-normalizations.

Reply:
With the latest changes, new sed rule changes other files too, we should reconsider this I think

Reply:
https://github.com/citusdata/citus/pull/3995/commits/045ef1d4b1b4c91a7c267c47166dd5438c663c65

Reply:
the other changes seem reasonable to me. I don't see how this normalization rule can be any harmful now or in the future, do you see anything?